### DISCUSSION TITLE
Disable vaultwarden admin token

### DISCUSSION THREAD

**Discussion by lukyjay** [tteck/Proxmox#2271] [Help Needed] [answered]
Hi

How do I disable the admin token in Vaultwarden? I've removed it from the config.json file and restarted the service but the admin panel still loads.

Thanks

**Comment by tteck** [comment, marked as answer]
`nano /opt/vaultwarden/.env`

**Reply by lukyjay** [reply]
Thanks! Just found it

There's also a token in /opt/vaultwarden/config.json

Perhaps this should be an option in the updater since there's no benefit (just risk) in leaving the admin panel active

**Reply by tteck** [reply]
It is using Argon2 to securely hash the password and is intended to be resistant to various types of attacks.
You will need the admin panel.

**Comment by OsipovOV** [comment]
I removed the allow new signups checkbox and checked that a new user cannot register. But after I disable the admin panel (commenting out the line admin_token..... in /opt/vaultwarden/config.json and /opt/vaultwarden/.env) the new user can register again. Is this like this for everyone? How to simultaneously disable new signups and admin panel?
